docs/database.md

```diff
@@ -6,31 +6,68 @@ MongoDB is een open source document-georiënteerde database en is geschreven in
 
 MongoDb is geschikt voor grote data het is schaalbaar en werk zonder relaties in tegenstelling tot bijvoorbeeld SQLite. Omdat we in ons project geen echte relaties nodig hebben is de keuze daarom ook makkelijk gemaakt voor MongoDB.
 
-### Voorbeeld eventuele database structuur
+## Database structuur
+
+### Collection polls
 
 ```json
 {
-  "users": {
-    "useridofmaliek": {
-      "username": "Maliek Meersschaert",
-      "password": "SOME HASH",
-      "polls": {
-        "1": {
-          "name": "Amerikaanse verkiezingen",
-          "options": {
-            "Donald Trump": 10,
-            "Hilary Clinton": 9,
-            "Gary Johnson": 2,
-            "Jill Stein": 1
-          }
-        }
-      }
+  "_id" : "1111c298aeb541d0776a9556",
```

Comment:
Dit is de mongo manier, die collections zijn eigenlijk zoals tabellen he. die _ids zijn ook door mongo gemaakt. en dat 2de voordeel is volgens mij heel belangrijk, zeker omdat polls ook anoniem kunnen zijn.